boot/zephyr/boards/b_u585i_iot02a.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Define the device, controller and partition to be the external memory
+ * for running the application in external NOR from MCUboot
+ */
+
+/delete-node/ &slot0_partition;
+/delete-node/ &slot1_partition;
+/delete-node/ &scratch_partition;
+/delete-node/ &storage_partition;
+
+&mx25lm51245 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		slot0_partition: partition@0 {
```

Comment:
as I said in the other PR, adding this here is pointless, this should go in your base board file in zephyr otherwise you will have completely different partitions for mcuboot vs every application, and the applications will not boot because of that

Reply:
I understand, that the slot 0 & 1 partitions are kept in the flash0
However, when building the samples/sysbuild/with_mcuboot/ --sysbuild, is there any  b_u585i_iot02a.overlay,   to declare the `zephyr,code-partition` to be placed in the external flash, not in the flash0 ?